core/src/main/scala/com/eed3si9n/jarjarabrams/Zip.scala

```diff
@@ -189,4 +190,23 @@ object Zip {
   def createDirectories(p: Path): Unit =
     if (Files.exists(p)) ()
     else Files.createDirectories(p)
+
+  def sha256(p: Path): String =
+    Using.fileInputStream(p) { in =>
+      val digest = MessageDigest.getInstance("SHA-256")
+      val hash = digest.digest(toByteArray(in))
```

Comment:
I wouldn't do this. What about:

https://docs.oracle.com/javase/8/docs/api/java/security/DigestInputStream.html

then use transfer with a no-op outputstream. No need to keep the whole file in memory just to hash.

Reply:
or just move it to the test code that uses it. I don't mind an inefficient implementation that is just for test.

Reply:
Addressed in 7204aea1b5b2610d85c09e66896b08bd4adbe5c6